### ISSUE TITLE
jtd-codegen generates invalid typescript if the empty string is allowed as an enum value

### ISSUE THREAD

**Issue by mlc** [jsontypedef/json-typedef-codegen#16]
Given the following input JTD:

```json
{
  "properties": {
    "a": {"type":  "string"},
    "b": {"enum":  ["", "x", "y"]}
  }
}
```

`jtd-codegen` with the `--typescript-out` flag generates the following output:
```typescript
// Code generated by jtd-codegen for TypeScript v0.2.0

export enum SimpleB {
   = "",
  X = "x",
  Y = "y",
}

export interface Simple {
  a: string;
  b: SimpleB;
}
```

However, the line ` = ""` with nothing before the equals sign is rejected by the typescript compiler as a syntax error.

`jtd-codegen` should either reject the input if I'm misunderstanding something and this is not valid JTD, or it should emit typescript which does not cause the typescript compiler to complain.

**Comment by mlc** [comment]
Thanks for the detailed response, @ucarion! We were able to work around the issue in our application (by modifying the schema), but it's good to hear that the "right" fix will be forthcoming as well!

**Comment by ucarion** [comment]
The fix for this should be incorporated into v0.4.1 of `jtd-codegen`. :)
